### PR TITLE
ENH: Copy epi acquisition metadata to sbref

### PR DESCRIPTION
**Pull request by effigies**
This PR copies the metadata (minus IntendedFor) from the `LR` fieldmap to the `LR` sbref, and likewise with `RL`.

### CODE REVIEW

**Comment by effigies** [comment]
@oesteban @mgxd Would appreciate review here. It's a bit less obviously correct than #1.

**Review by mgxd — APPROVED** [review]
Should be fine - if we have access to the DICOMs, we can check to be 100% but generally they match.

One quick catch - are the `sbref`s in the same orientation as the `bold`s?

File: sub-101006/func/sub-101006_task-rest_dir-LR_sbref.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "PhaseEncodingDirection": "i",
```
Comment (oesteban) [inline sub-101006/func/sub-101006_task-rest_dir-LR_sbref.json:2]: Is the dataset oriented Rxx? -- other than that, this is okay.

I'm not 100% positive I haven't tinkered with the BOLDs orientation/PEs - this PR is a good opportunity to check I didn't mess up.

**Review by oesteban — APPROVED** [review]

**Comment by effigies** [comment]
We don't have access to the DICOMs, AFAIK. This is converted from published HCP data, which I think is all NIfTI. But yes, everything is LAS.

**Comment by oesteban** [comment]
If it's LAS, then either the `dir-LR` entity or the `PhaseEncodingDirection` metadata are messed up, right?

**Comment by oesteban** [comment]
BTW, an alternative way of checking this would be accessing this subject through the DataLad layer. I remember they released a BIDS-formatted version of HCP. I don't know how they get around the problem of access though.

**Comment by effigies** [comment]
I thought so as well, but it's consistent across fieldmaps, sbrefs and bold, as well as consistent with HCP's labeling. I think, they switched the `LR` and `RL` labels in the filenames, but the `"i"` and `"i-"` look consistent with the data itself.

**Comment by oesteban** [comment]
Good to hear I didn't mess up with those. Would it be too much to ask adding a note about this conversation on the README? At least say what you mention that labels seem confused.

**Comment by effigies** [comment]
Merging. Feel free to clean up the wording later.

**Pull request merged**